experiment/build/docker_images.py

```diff
@@ -40,7 +44,29 @@ def _instantiate_image_obj(name_template, obj_template, fuzzer, benchmark):
 def _get_image_type_templates():
     """Loads the image types config that contains "templates" describing how to
     build them and their dependencies."""
-    return yaml_utils.read('docker/image_types.yaml')
+    all_templates = yaml_utils.read('docker/image_types.yaml')
+    return all_templates
+
+
+def get_fuzzers_and_benchmarks():
```

Comment:
use get_all_benchmarks from benchmark_utils and get_fuzzer_names from fuzzer_utils instead of this function please.